### DISCUSSION TITLE
PlantVillage

### DISCUSSION THREAD

**Discussion by aymen3110** [huggingface/datasets#4136] [Q&A]
hi can you add dataset plant village

**Comment by mariosasko** [comment]
Hi! You can use `imagefolder` to load this dataset after cloning the dataset repo from `git clone git@github.com:spMohanty/PlantVillage-Dataset.git`:
```python
from datasets import load_dataset
dset = load_dataset("imagefolder", data_dir="PlantVillage-Dataset/color", ignore_verifications=True) # relative path to data dir
```

**Reply by aymen3110** [reply]
hi ik that but i want a dataset like beans format csv
